src/Libraries/Microsoft.Extensions.AI/JsonDefaults.cs

```diff
@@ -30,9 +31,10 @@ private static JsonSerializerOptions CreateDefaultOptions()
             // Keep in sync with the JsonSourceGenerationOptions on JsonContext below.
             var options = new JsonSerializerOptions(JsonSerializerDefaults.Web)
             {
-                WriteIndented = true,
-                DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
                 TypeInfoResolver = new DefaultJsonTypeInfoResolver(),
+                Converters = { new JsonStringEnumConverter() },
```

Comment:
Are there any negative implications to this?

Reply:
I can't think of any reason why one might want to serialize enums as integers.